src/components/common/Footer/index.tsx

```diff
@@ -129,7 +129,7 @@ const Footer: React.VFC = () => {
         dangerouslySetInnerHTML={{ __html: settings['footer.custom.html'] }}
       />
     )
-  } else if (settings['footer.type'] === 'multiline') {
+  } else if (settings['footer.type'] === 'multiple') {
```

Comment:
NITS: check database settings value which is `multiline`, if you changed the comparison value to `multiple`